src/App.jsx

```diff
@@ -98,139 +109,138 @@ function App() {
     }
     setCityPart(cityPart);
     if (cityPart.length >= 2) {
-      const filteredCities = cities.filter(c => !getGuesses(history, todaySeed).includes(c));
-      const matchedCities = new Set(
-        [...filteredCities.filter(c => normalize(c.name).startsWith(normalize(cityPart.trim()))),
-          ...filteredCities.filter(c => normalize(c.name).includes(normalize(cityPart.trim())))]
-      );
+      const filteredCities = cities.filter((c) => !getGuesses(history, todaySeed).includes(c));
+      const matchedCities = new Set([
+        ...filteredCities.filter((c) => normalize(c.name).startsWith(normalize(cityPart.trim()))),
+        ...filteredCities.filter((c) => normalize(c.name).includes(normalize(cityPart.trim())))
+      ]);
```

Comment:
@filipsakala nevíš jak nastavit formatter, aby sem nedával středníky? Nevypadá to pak jako java :D